pkgs/top-level/python-packages.nix

```diff
@@ -7317,6 +7332,23 @@ in modules // {
     };
   };
 
+  jsbeautifier = buildPythonApplication rec {
+    name = "jsbeautifier-1.6.4";
+
+    buildInputs = with self; [ editorconfig pytest six ];
```

Comment:
Are you sure it needs six only during building, and not during run-time?
Also, did you check whether the tests are actually run with all interpreter versions?